project.clj

```diff
@@ -13,21 +13,28 @@
   ;; requires lein 2.2.0+.
   :pedantic? :abort
 
+  :exclusions [org.clojure/clojure]
```

Comment:
lein pedantic will abort because `ring-middleware`, `http-client` (a transitive), and `clj-time` require Clojure 1.7. It then recommends doing an exclusion on those libraries for Clojure.

I don't know why the explicit reference to Clojure doesn't already override them, but either bumping the Clojure dependency here, adding exclusions to those libraries, or adding a global exclusion is required to satisfy lein pedantic.

I went with the global exclusion because I thought bumping the Clojure version might be considered a breaking change and I had seen another project do it. I don't know if that makes it right though.